dealer.py

```diff
@@ -0,0 +1,8 @@
+from player import Player
```

Comment:
Making this it's own file is good. We probably want to keep it with he player stuff though. We can turn `player` into a directory to hold all the types of players. Then in the `__init__.py` file in `player` we can use `__all__` to make it easier to import all the objects. We can talk about it when we pair.